### ISSUE TITLE
no internet

### ISSUE THREAD

**Issue by AndroEhab** [trailofbits/algo#14424]
I did all the installations and got the congratulations message but soon as I connect I have no internet. I am on windows 10 and used WSL and digital ocean

**Comment by ltz0120** [comment]
I got the same problem

**Comment by MGelbana-Incorta** [comment]
This happens to me but after a few hours or may be right after the installation is over 😞 
I'm using DigitalOcean and I tried this many times, same result!

**Comment by jpcaparas** [comment]
I had the same issue, and I tried various troubleshooting steps on a Droplet (Ubuntu v20) in Sydney:

1. Reducing the MTU threshold
2. Changing the Wireguard port to 443
3. Looking at DigitalOcean firewall rules
4. Checking `iptables` rules

In the end, I just spun up a new Droplet in a different region (Singapore) using Ubuntu v22 (LTS) and bootstrapped Wireguard again, and it worked right off of the bat. 

TLDR: The Droplet was the issue.
